lib/split/experiment.rb

```diff
@@ -192,27 +207,23 @@ def random_alternative
     end
 
     def version
-      @version ||= (redis.get("#{name}:version").to_i || 0)
+      @version ||= (redis.get("#{name}:version").to_i || 0) rescue @version ||= 0
```

Comment:
What's the exception we're swallowing here?

Reply:
removing this